machineconfiguration/v1alpha1/tests/machineosbuilds.machineconfiguration.openshift.io/OnClusterBuild.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "[TechPreview] MachineOSBuild"
+crdName: 0000_80_machine-config_01_machineosbuilds-TechPreviewNoUpgrade.crd.yaml
```

Comment:
rdname should be the name of the CRD, not the filename.  machineosbuilds.machineconfiguration.openshfit.io I bet?